schedule.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles/styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Oswald&display=swap" rel="stylesheet">    <title>BCIT - Schedule</title>
+</head>
+<body>
+    <header>
+        <div class="header-content">
+            <h1><a href="index.html">BCIT</a></h1>
+            <button class="btn-menu">Menu</button>
+            <nav>
+                <ul>
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="course.html">Course</a></li>
+                    <li><a href="schedule.html">Schedule</a></li>
+                    <li><a href="lessons.html">Lessons</a></li>
+                    <li><a href="contact.html">Contact</a></li>
+                    <li><a href="login.html">Login</a></li>
+                </ul>
+            </nav>	
+        </div>
+    </header>
+
+   <div class="wrapper">
+       <main>
+           <div class="top-banner">
+                <div class="banner-wrapper">
+                    <p class="title">Weekly Schedule</p>
+                </div>
+            </div>
+
+           <div class="mid-banner">
+               <div class="container">
```

Comment:
Please take a look at the example: https://comp1950.michaelwhyte.ca/final-project-student-samples/final-project-example-01/schedule.php
Instead of the list of lessons here should be the list of days.

Reply:
Cool, I'll update it